### ISSUE TITLE
Additional Phaser Cannons (Pulse phasers?)

### ISSUE THREAD

**Issue by ZayntIcarus** [Thorium-Sim/thorium#1867]
As a note, I have several ideas for small-scale weapon systems (Not THX, or fighter bay sized weapons) for thorium, as there are more than just torpedoes and phasers as weapons throughout science fiction that I believe we could incorporate into thorium. To minimize the amount of just random ideas that I have, I am making sure to include graphics and (hopefully) detailed descriptions on how I believe the systems would work. I hope this will also help reduce the amount of work on your end to imagine how these systems would look, assuming you eventually choose to add them into thorium.
(To contact me with any questions, please email me at: hexearcan@gmail.com, or contact me through discord: Zaynt Icarus#1300)

## Detailed Description
I believe we could somewhat incorporate an additional phaser cannon system into thorium, so other ships could have "alternate" phaser systems, similar to how the USS Phoenix has pulse phasers in their current Mercury controls. 
To do this, thorium would need to allow the ability to equip more than just one phaser system in the systems config when creating a simulator. When added, on the phaser charging screen/card, the ability to change the degree of the phaser blast would be removed, and an extra phaser cannon control would appear beneath the current, possibly with a scroll bar depending on the size of the screen. Something like this:
![pulsephas](https://user-images.githubusercontent.com/47375350/53317581-72404380-3889-11e9-818a-4293d310942a.png)

I understand that you can not have a phaser charging screen and instead use targeting, I believe a somewhat similar concept would be used, where they would again have to scroll down in the area where they charge the phasers, so they are able to separately charge the second set of cannons. Although preferably if you were to add more than the standard one phaser system it means you have a reason to also add the phaser charging card.

-- Mason P.

**Comment by alexanderson1993** [comment]
Can you clarify two things that aren't apparently obvious:

1. How does this help the crew have a better experience?
2. How does this help the flight director tell better stories?

As a seasoned Phoenix flight director, the pulse phasers were always kind of gimmicky and didn't really have any differentiation with the regular phasers.

**Comment by ZayntIcarus** [comment]
Good point, my apologies for not stating those earlier. Thinking about it further, I realize that it would only provide the option for flight directors with another set of phaser cannons, so how that would help flight directors would really be up to them and how they tell the story. 
That being said, they don't really help the director tell a better story, and there's a decent chance the crew wouldn't have a better or worse experience with it. My apologies for wasting some of your time.

**Comment by alexanderson1993** [comment]
No time wasted yet - I haven't worked on the feature. Having good justification helps me prioritize what features should be implemented sooner. Something like this might be useful if Thorium were to track HP for targets - phasers do less damage than pulse phasers. Something like that.
